composer.json

```diff
@@ -19,7 +19,7 @@
         "laminas/laminas-eventmanager": "<3.4.0",
         "laminas/laminas-i18n": "<2.12.0",
         "laminas/laminas-recaptcha": "<3.4.0",
-        "laminas/laminas-servicemanager": "<3.10.0",
+        "laminas/laminas-servicemanager": "<3.14.0",
```

Comment:
To switch without a BC break, `laminas/laminas-servicemanager` must be in the `require` section

Reply:
I thought the conflict section would take care of that? Say for instance, users on mezzio using a different container implementation, but still currently using laminas-form, this would require them to use laminas-servicemanager.

Adding the conflict for `<3.14.0` should prevent anyone from being able to use this latest version with an incompatible service manager, without specifically requiring it?